### PR TITLE
Optimize bundle

### PR DESCRIPTION
**Pull request by juanca**
**Summary**

**Easy** wins in optimizing.

Noticed the bundle was approximately 400KB! Clearly our external libraries was taking up much of the space. After removing jQuery and some lodash libraries, this is reporting a 16BK bundle! By extracting the dependencies as peer dependencies, host modules can optimize their libraries as needed! 

**Test plan**

Bundle before:

screen shot 2017-09-18 at 3 14 34 pm

Bundle after:

screen shot 2017-09-18 at 3 15 46 pm

- [ ] Soon

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,14 +56,13 @@
...
-    "lodash.find": "^4.6.0",
-    "lodash.isobject": "^3.0.2",
-    "lodash.omit": "^4.5.0",
-    "lodash.pickby": "^4.6.0"
+    "lodash.isobject": "^3.0.2"
   },
   "peerDependencies": {
     "brainstem-js": "^0.4.30",
     "jquery": "2.2.4",
+    "lodash.find": "^4.6.0",
```
Comment (naganowl) [inline package.json]: This can be used off Underscore and possibly even native as well.

**Comment by juanca** [comment]
@naganowl I went full native since both methods were easily replaced. 

Thinking this is enough for an optimized bundle. I made an issue to track efforts in nailing down our bundling strategy. 

Also, I made a change in `master` about checking the example. It is finally working in this PR since it hasn't been given love in a looong while.

File: package.json [hunk 2]
```diff
@@ -89,12 +86,14 @@
     "karma-phantomjs-launcher": "^1.0.2",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.8.0",
+    "lodash.omit": "^4.5.0",
```
Comment (naganowl) [inline package.json]: Underscore has omit which can be used instead of pulling this in.

Comment (juanca) [reply package.json]: We don't have underscore in this library.

Comment (naganowl) [reply package.json]: Touche

**Comment by juanca** [comment]
https://github.com/mavenlink/mavenlink/pull/9515

**Pull request merged**